src/components/Popover/Popover.vue

```diff
@@ -74,6 +75,24 @@ export default {
 	components: {
 		VPopover,
 	},
+
+	mounted() {
+		this.$watch(
+			() => {
+				// required because v-tooltip doesn't provide events
+				// and @show is too early
+				// see https://github.com/Akryum/v-tooltip/issues/661
+				return this.$refs.popover.isOpen
+			},
+			(val) => {
+				if (val) {
+					this.$emit('after-show')
```

Comment:
Please document those events

Reply:
@PVince81 you can document them for our styleguide directly, no need for a readme :)
https://vue-styleguidist.github.io/docs/Documenting.html#events